src/types/entities/agents/manager.ts

```diff
@@ -102,12 +102,16 @@ export interface AgentManager {
      * Method to close all connections with agent, stream and web socket
      */
     disconnect: () => Promise<void>;
+    /**
```

Comment:
wouldnt it be easier and cleaner to do some like

chat: (userMessage: string, chatMode: chat-mpode,append_chat?: boolean)

within the same methos?

Reply:
we wanted to handle chatMode only in agents-ui, so we separate everything in agent-sdk so things won't get mixed up and only in agents-ui I have this differentiation by chatMode.

Reply:
gotchta - this is sdk :)